R/params.R

```diff
@@ -48,31 +48,54 @@ params_label <- function(inputControlFn, param) {
   label
 }
 
-params_value_to_ui <- function(inputControlFn, value) {
+params_value_to_ui <- function(inputControlFn, value, showDefault) {
```

Comment:
Would it be valuable to have two separate functions: 
* given an `inputControlFn`, return a "default" value for that control.
* given an `inputControlFn`, massage a value appropriately for that control (the current implementation).
this separation might make for easier-to-grok functions and simpler control structure in each (though some repetition).